docs/development.md

```diff
@@ -58,9 +58,7 @@ cd foreman
 bundle exec foreman start
 ```
 
-### Webpack dev server
-
-When using the dev server you need to accept the self-signed certificate. You should go to `https://centos7-devel.example.com:3808` (or your equivalent) and add an exception for this certificate. 
+The Foreman UI is then accessible on `https://centos7-devel.<hostname>.example.com/`, where `<hostname>` is the shortname of your workstation.  While using the webpack dev server, you must use HTTPS and the hostname, as well as visiting `https://centos7-devel.<hostname>.example.com:3808` to accept the self-signed certificate.
 
 ### Reviewing Pull Requests
```

Comment:
I'm kind of curious why we mention this, this seems like a very specific workflow used by one developer, is there a benefit to rpr over using `hub checkout <PR url>`? 

But it's only part of the foreman devel box, not katello. Should we update it and include this rpr script there too? Or can we recommend something more mainstream like hub?

Any thoughts @ares?

Reply:
I use https://gist.github.com/piscisaureus/3342247 method so we have a few floating out there (I ran into too many issues with hub in my past). To the question of whether this should live here or not, during the original PR we discussed this. The take aways for another day (maybe this is the another day day!) were:

 1) Should we centralize on single workflows or provide docs for multiple
 2) Where should tools and docs for development live? Should we create a dev-tools repository? Should we set that up by default? Should we just put it all here in Forklift?